lib/flipper/views/features.erb

```diff
@@ -0,0 +1,35 @@
+<div class="vads-l-row">
+  <div class="vads-l-col">
+    <table class="vads-u-margin-top--0">
+      <caption class="vads-u-font-size--xl">Features</caption>
+      <thead>
+        <tr>
+          <th>Status</th>
+          <th>Feature</th>
+          <th>Description</th>
+          <th>Enabled Gates</th>
```

Comment:
Kept this the same for now, but I'm sure there's a better name for this to describe the conditions for enabled features...

Reply:
This is fine with me.